denizen_scripts/survival/repo-link/command_feed.dsc

```diff
@@ -0,0 +1,18 @@
+feed_command:
+    type: command
+    name: feed
+    usage: /feed (player)
+    description: feeds a player
+    permission: adriftus.staff
+    script:
+    - if <context.args.size> > 1:
+        - inject command_syntax
+    - if <context.args.is_empty>:
+        - feed 20 <player>
+        - narrate "You have been fed."
+    - else:
+        - define user <context.args.first>
+        - inject player_verification
+        - feed 20 <[user]>
```

Comment:
The syntax for this command is `feed (<entity>) (amount:<#>) (saturation:<#.#>)`
```suggestion
        - feed amount:20 <[user]>
```